node/src/main/scala/com/wavesplatform/utx/UtxPoolImpl.scala

```diff
@@ -173,6 +156,23 @@ class UtxPoolImpl(time: Time, blockchain: Blockchain, spendableBalanceChanged: O
     Option(transactions.remove(txId))
       .foreach(afterRemove)
 
+  private[this] def addTransaction(tx: Transaction, verify: Boolean): TracedResult[ValidationError, Boolean] = {
+    if (!transactions.containsKey(tx.id())) {
+      val diff = TransactionDiffer(blockchain.lastBlockTimestamp, time.correctedTime(), blockchain.height, verify)(blockchain, tx)
+      diff.resultE.foreach {
+        PoolMetrics.addTransaction(tx)
```

Comment:
Pool metrics should be incremented even when diff is `Left`, but `verify` is `True`.